Page transitions between home hero and nav

chadcrume opened the pull request.

alvarix commented:
@chadcrume interesting! 
This wasnt quite the brief though: 
Allow the logo to transition from a large hero element on home to a small navbar element elsewhere.

This is pretty exciting but I would put this in a category of a different theme option. 
A little too much movement for my taste on every page change. 
Another note, in this variation it would be good to expose a prominent: _User prefers reduced motion_ button to disable the animations. 

WDYT

chadcrume commented:
@alvarix , yes I got carried away just picking up the navigation transitions from your site, and then some. I've changed it to be on spec. Also removed color from logos (footer too) and a couple of other tweaks.
